msibi/optimize.py

```diff
@@ -120,15 +120,9 @@ def optimize(
         if engine == "hoomd":
             try:
                 import hoomd
-
                 HOOMD_VERSION = 2
             except ImportError:
-                try:
-                    import hoomd_script
-
-                    HOOMD_VERSION = 1
-                except ImportError:
-                    raise ImportError("Cannot import hoomd")
+                raise ImportError("Cannot import hoomd")
```

Comment:
I think we can change this to 
```suggestion
        if engine == "hoomd":
            import hoomd
            HOOMD_VERSION = 2
        else:  # don't need a hoomd version if not using hoomd
            HOOMD_VERSION = None
```
We don't need the try except if we just raise the exception anyway.

Reply:
Also sorry, the "suggestion" feature on lines with addition/deletion is really wonky.

Reply:
Ok, made these changes if you want to double check.